R/utils_cypress.R

```diff
@@ -12,28 +12,43 @@ create_cypress_structure <- function(app_dir, port, debug) {
   dir_tests <- tempdir()
 
   # node path
-  node_path <- file.path(dir_tests, "node")
-  root_path <- file.path(node_path, "root") # nolint
+  node_path <- fs::path(dir_tests, "node")
+  root_path <- fs::path(node_path, "root") # nolint
 
   # test path
-  tests_path <- file.path(dir_tests, "tests")
-  cypress_path <- file.path(tests_path, "cypress")
-  integration_path <- file.path(cypress_path, "integration")
-  plugins_path <- file.path(cypress_path, "plugins")
+  tests_path <- fs::path(dir_tests, "tests")
+  cypress_path <- fs::path(tests_path, "cypress")
+  integration_path <- fs::path(cypress_path, "integration")
+  plugins_path <- fs::path(cypress_path, "plugins")
 
   # creating paths
-  dir.create(path = node_path, showWarnings = FALSE)
-  dir.create(path = tests_path, showWarnings = FALSE)
-  dir.create(path = cypress_path, showWarnings = FALSE)
-  dir.create(path = integration_path, showWarnings = FALSE)
-  dir.create(path = plugins_path, showWarnings = FALSE)
+  fs::dir_create(path = node_path)
+  fs::dir_create(path = tests_path)
+  fs::dir_create(path = cypress_path)
+  fs::dir_create(path = integration_path)
+  fs::dir_create(path = plugins_path)
 
   # create a path root linked to the main directory app
-  file.symlink(from = app_dir, to = root_path)
+  tryCatch(
+    expr = {
+      fs::link_create(app_dir, root_path, symbolic = TRUE)
+    },
+    error = function(e) {
+      # If system cannot symlink then try to clone the repository
+      #  This may happen on some windows versions
+      #  This can be an expensive operation on big repositories
```

Comment:
What do you think about adding a prompt question if the user really wants to perform it (since it is a big operation)?

Reply:
Will do!

Do you want to debate if it's even necessary. Reaching this exception will be very rare now that we're using the `fs` package